src/panels/dev-info/system-log-card.ts

```diff
@@ -68,6 +68,16 @@ class SystemLogCard extends LitElement {
                                 this.hass!.language
                               )}
                               ${item.source} (${item.level})
+                              ${item.count > 1
```

Comment:
These properties need to be added to the type https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/data/system_log.ts#L3-L10